client/src/app/site/chat/components/chat-group-list/chat-group-list.component.html

```diff
@@ -0,0 +1,53 @@
+<os-head-bar [hasMainButton]="hasMainButton" (mainEvent)="createChatGroup()">
+    <!-- Title -->
+    <div class="title-slot">
+        {{ 'Chat' | translate }}
+    </div>
+</os-head-bar>
+
+<mat-card
+    *ngIf="chatGroupsObservable | async as chatGroups"
+    class="os-card chat-group-list-card"
+    [ngClass]="{ 'fit-height': !chatGroups?.length }"
+>
+    <div class="padding-16" *ngIf="!chatGroups?.length">
+        <span>
+            {{ 'No chat groups available.' | translate }}
+        </span>
+    </div>
+    <os-vertical-tab-group
+        *ngIf="chatGroups?.length"
+        [labelHeight]="56"
+        [showAlwaysContentHeader]="true"
+        (contentStateChanged)="onTabContentChanged($event)"
+    >
+        <ng-template osVerticalTabGroupLabelHeader>
+            <div
+                class="flex-vertical-center padding-left-16 padding-right-16 full-width full-height icon fake-disabled"
+            >
```

Comment:
the directive `osVerticalTabGroupLabelHeader` is completely empty and can probably be deleted

Reply:
The component `VerticalTabGroupComponent` reads the `<ng-template />` by this directive.